src/Goerli-DssSpell.sol

```diff
@@ -20,72 +20,79 @@ pragma experimental ABIEncoderV2;
-    address constant MCD_CLIP_CALC_WSTETH_A   = 0xb4f2f0eDFc10e9084a8bba23d84aF2c23B312852;
+    // --- Rate ---
+    uint256 constant SEVEN_PCT_RATE         = 1000000002145441671308778766;
 
-    function actions() public override {
+    // --- Math ---
+    uint256 constant MILLION = 10**6;
 
-        // GUNIV3DAIUSDC-A Parameter Adjustments
-        // https://vote.makerdao.com/polling/QmemHGSM?network=mainnet
-        // https://forum.makerdao.com/t/request-to-raise-the-guniv3daiusdc1-a-dc-to-500m/11394
-        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC1-A", 500 * MILLION);     // Set DCIAM Max debt ceiling to 500 M
-        DssExecLib.setIlkLiquidationRatio("GUNIV3DAIUSDC1-A", 10200);                // Set LR to 102 %
-        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_FIVE_PCT_RATE, true); // Set stability fee to 0.5 %
+    // --- WBTC-B ---
+    address public immutable WBTC;
+    address public immutable PIP_WBTC;
+    address constant MCD_JOIN_WBTC_B        = 0x13B8EB3d2d40A00d65fD30abF247eb470dDF6C25;
+    address constant MCD_CLIP_WBTC_B        = 0x4F51B15f8B86822d2Eca8a74BB4bA1e3c64F733F;
+    address constant MCD_CLIP_CALC_WBTC_B   = 0x1b5a9aDaf15CAE0e3d0349be18b77180C1a0deCc;
 
-        // Add WSTETH-A as a new Vault Type (It should have come on version 1.9.8)
-       DssExecLib.addNewCollateral(CollateralOpts({
-            ilk:                   "WSTETH-A",
-            gem:                   WSTETH,
-            join:                  MCD_JOIN_WSTETH_A,
-            clip:                  MCD_CLIP_WSTETH_A,
-            calc:                  MCD_CLIP_CALC_WSTETH_A,
-            pip:                   PIP_WSTETH,
-            isLiquidatable:        true,
-            isOSM:                 true,
-            whitelistOSM:          true,
-            ilkDebtCeiling:        3 * MILLION,
-            minVaultAmount:        10000,
-            maxLiquidationAmount:  3 * MILLION,
-            liquidationPenalty:    1300,        // 13% penalty fee
-            ilkStabilityFee:       FOUR_PCT_RATE,
-            startingPriceFactor:   13000,       // Auction price begins at 130% of oracle
-            breakerTolerance:      5000,        // Allows for a 50% hourly price drop before disabling liquidations
-            auctionDuration:       140 minutes,
-            permittedDrop:         4000,        // 40% price drop before reset
-            liquidationRatio:      16000,       // 160% collateralization
-            kprFlatReward:         300,         // 300 Dai
-            kprPctReward:          10           // 0.1%
-        }));
-        DssExecLib.setStairstepExponentialDecrease(MCD_CLIP_CALC_WSTETH_A, 90 seconds, 9900);
-        DssExecLib.setIlkAutoLineParameters("WSTETH-A", 50 * MILLION, 3 * MILLION, 8 hours);
+    constructor() public {
+        WBTC = DssExecLib.getChangelogAddress("WBTC");
+        PIP_WBTC = DssExecLib.getChangelogAddress("PIP_WBTC");
+    }
```

Comment:
The immutable thing is actually a good hack in case we get stack too deep. We will definitely need it if NEXO thing is happening. Otherwise we could just get them as regular variables inside the `actions()` (the regular thing we do).
Anyway let's leave them as they are for now, if we confirm that the spell won't be that big we can do the change.